### PR TITLE
Customer manager

### PR DESCRIPTION
**Pull request by qiaoruzhang**
Update customer, customer menager, create Membership. And all there test files. All tests passed.

### CODE REVIEW

File: src/main/java/Customer.java [hunk 1]
```diff
@@ -9,6 +9,7 @@
     private int balance;
     private int millage;
```
Comment (oliviaj0422) [inline src/main/java/Customer.java]: "millage" should be mileage

File: src/main/java/Membership.java [hunk 2]
```diff
@@ -22,6 +22,21 @@ public boolean checkMembership(Customer customer) {
...
     }
+    /**
+     * Check this customer's current membership status
+     *
+     * @param customer The customer needed to Check current membership status.
+     */
+    public int checkMembershiplevel(Customer customer) {
+        if (customer.checkMembership()){
+            if (this.MembershipCustomer.containsKey(customer.getUsername())) {
+                return customer.checkMembershiplevel();}
```
Comment (oliviaj0422) [inline src/main/java/Membership.java:33]: lines 33 and 36 and 38 are redundant. May be reduced to one line.

**Review by oliviaj0422 — CHANGES_REQUESTED** [review]
Well done overall

File: src/main/java/Membership.java (hunk 2 above)
Comment (qiaoruzhang) [reply src/main/java/Membership.java:33]: for double check purposes, but could remove

File: src/main/java/Customer.java (hunk 1 above)
Comment (qiaoruzhang) [reply src/main/java/Customer.java]: Thank you boss

**Review by qiaoruzhang — COMMENTED** [review]
Okay

**Comment by Jasmine-Zhuang** [comment]
Could you write a method for redeem points decrement?

**Pull request closed without merging**